spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/ops/RankSpec.scala

```diff
@@ -24,7 +24,7 @@ class RankSpec extends FlatSpec with Matchers {
     val input =
         Tensor[Float](T(1f, 2f, 2f))
 
-    val expectOutput = Tensor[Int](T(1))
+    val expectOutput = Tensor[Int](Array(1), Array[Int]())
```

Comment:
should we create some Scalar factory method for user to use. So user can write something like

```scala
val output = Scala(1)
```

Reply:
Do you mean
```scala
Tensor.scalar(1)
```
Or create a scalar object?